grape.gemspec

```diff
@@ -19,7 +19,9 @@ Gem::Specification.new do |s|
   s.add_runtime_dependency 'rack-accept'
   s.add_runtime_dependency 'virtus', '>= 1.0.0'
 
-  s.files         = Dir['**/*'].keep_if { |file| File.file?(file) }
+  s.files         = %w[CHANGELOG.md CONTRIBUTING.md README.md UPGRADING.md LICENSE]
+  s.files        += %w[Rakefile grape.gemspec]
```

Comment:
it seems to me that those two are not needed for the distribution

Reply:
Don't you need .gemspec? How does it know dependencies otherwise?

Reply:
You do need .gemspec for dependencies. That's the actual declaration of the gem, and you would get "Could not find gem 'grape' in source at `vendor/grape-1.2.5`." if you `gem unpack` what you build without it and try to `bundle install`.

Reply:
I do not need it, for sure :) this file is not a requirement for the gem distribution, though the workflow with "gem unpack" you mentioned above requires it because `path` option expects either [`gemspec` file or explicit `version` of the gem](https://bundler.io/man/gemfile.5.html#PATH)(the mentioned issue could be fixed by using `gem 'grape', '1.2.5', path: 'vendor/grape-1.2.5'`).

Anyway, I'm okay with that